cli/src/tasks/migrate.ts

```diff
@@ -0,0 +1,734 @@
+  //*
+
+  allDependencies = {
+    ...config.app.package.dependencies,
+    ...config.app.package.devDependencies,
+  };
+
+  const monorepoWarning =
+    'Please note this tool is not intended for use in a mono-repo enviroment, please check out the Ionic vscode extension for this functionality.';
+
+  logger.info(monorepoWarning);
+
+  const { migrateconfirm } = await logPrompt(
+    `Capacitor 4 sets a deployment target of iOS 13 and Android 12 (SDK 32). \n`,
+    {
+      type: 'text',
+      name: 'migrateconfirm',
+      message: `Are you sure you want to migrate? (Y/n)`,
+      initial: 'y',
+    },
+  );
+
+  if (
+    typeof migrateconfirm === 'string' &&
+    migrateconfirm.toLowerCase() === 'y'
+  ) {
+    try {
+      const { npmInstallConfirm } = await logPrompt(
+        `Would you like the migrator to run npm install to install the latest versions of capacitor packages? (Those using other package managers should answer N)`,
+        {
+          type: 'text',
+          name: 'npmInstallConfirm',
+          message: `Run Npm Install? (Y/n)`,
+          initial: 'y',
+        },
+      );
+      const runNpmInstall =
+        typeof npmInstallConfirm === 'string' &&
+        npmInstallConfirm.toLowerCase() === 'y';
+
+      await runTask(`Installing Latest NPM Modules.`, () => {
+        return installLatestNPMLibs(runNpmInstall, config);
+      });
+
+      await runTask(
+        `Migrating @capacitor/storage to @capacitor/preferences.`,
+        () => {
+          return migrateStoragePluginToPreferences(runNpmInstall);
+        },
+      );
+
+      if (
+        allDependencies['@capacitor/ios'] &&
+        existsSync(config.ios.platformDirAbs)
+      ) {
+        // Set deployment target to 13.0
+        await runTask(`Migrating deployment target to 13.0.`, () => {
+          return updateFile(
+            config,
+            join(config.ios.nativeXcodeProjDirAbs, 'project.pbxproj'),
+            'IPHONEOS_DEPLOYMENT_TARGET = ',
+            ';',
+            '13.0',
+          );
+        });
+
+        // Update Podfile to 13.0
+        await runTask(`Migrating Podfile to 13.0.`, () => {
+          return updateFile(
+            config,
+            join(config.ios.nativeProjectDirAbs, 'Podfile'),
+            `platform :ios, '`,
+            `'`,
+            '13.0',
+          );
+        });
+
+        // Remove touchesBegan
+        await runTask(
+          `Migrating AppDelegate.swift by removing touchesBegan.`,
+          () => {
+            return updateFile(
+              config,
+              join(config.ios.nativeTargetDirAbs, 'AppDelegate.swift'),
+              `override func touchesBegan`,
+              `}`,
+            );
+          },
+        );
+
+        // Remove NSAppTransportSecurity
+        await runTask(
+          `Migrating info.plist by removing NSAppTransportSecurity key.`,
+          () => {
+            return removeKey(
+              join(config.ios.nativeTargetDirAbs, 'info.plist'),
```

Comment:
```suggestion
              join(config.ios.nativeTargetDirAbs, 'Info.plist'),
```

Info.plist is uppercase, this might cause problems in case sensitive OS